root/service_worker.js

```diff
@@ -0,0 +1,77 @@
+var cacheName = "<% version %>";
+var offlineFile = "/";
+var filesToCache = [
+    offlineFile,
+    '/static/manifest.json',
+    <%- FOREACH css IN req.env.item('psgix.assets').grep(/\.css$/) %>
+    "<% css %>",
+    <%- END %>
+    "/static/opensearch.xml",
+    "/static/icons/favicon.ico",
+    "/static/icons/apple-touch-icon.png",
+    <%- FOREACH js IN req.env.item('psgix.assets').grep(/\.js$/) %>
+    "<% js %>",
+    <%- END %>
+    <%- FOREACH less IN req.env.item('psgix.assets').grep(/\.less$/) %>
+    "/static/less/style.less",
+    <%- need_less = 1 %>
+    <%- END %>
+    <%- IF need_less %>
+    "/static/js/less.min.js"
+    <%- END %>
+];
```

Comment:
I'm sure I am missing some resources that can be precached.

Reply:
The fonts?